SelfSignedCertificate/SelfSignedCertificate.m

```diff
@@ -23,27 +23,33 @@
   "\x57\x93\x3e\xcb\x93\xff\x6e\xde\xd1";
 
 const unsigned char *cert = (unsigned char*)
-  "\x30\x82\x01\x15\x30\x81\xbd\xa0\x03\x02\x01\x02\x02\x01\x01\x30"
-  "\x0a\x06\x08\x2a\x86\x48\xce\x3d\x04\x03\x02\x30\x15\x31\x13\x30"
-  "\x11\x06\x03\x55\x04\x03\x0c\x0a\x6d\x61\x73\x74\x61\x68\x79\x65"
-  "\x74\x69\x30\x1e\x17\x0d\x31\x37\x30\x36\x30\x39\x31\x34\x30\x38"
-  "\x31\x37\x5a\x17\x0d\x31\x37\x30\x36\x31\x30\x31\x34\x30\x38\x31"
-  "\x37\x5a\x30\x15\x31\x13\x30\x11\x06\x03\x55\x04\x03\x0c\x0a\x6d"
-  "\x61\x73\x74\x61\x68\x79\x65\x74\x69\x30\x59\x30\x13\x06\x07\x2a"
-  "\x86\x48\xce\x3d\x02\x01\x06\x08\x2a\x86\x48\xce\x3d\x03\x01\x07"
-  "\x03\x42\x00\x04\xf6\x9c\xab\x24\x14\x4b\xb4\xef\x87\xf7\x0f\x23"
-  "\x1c\x5c\xd4\xf5\x78\x04\xac\xf8\xe0\xc6\xb2\xb3\xe3\x52\x18\x3d"
-  "\x80\x39\x1f\x6b\xd2\x79\xd2\x6a\x4c\x83\x64\x74\xe6\xc2\xda\x23"
-  "\x93\xff\xac\x1d\x50\x34\x6c\x5c\x23\x90\x65\x57\x93\x3e\xcb\x93"
-  "\xff\x6e\xde\xd1\x30\x0a\x06\x08\x2a\x86\x48\xce\x3d\x04\x03\x02"
-  "\x03\x47\x00\x30\x44\x02\x20\x7c\xa5\x9b\x1e\x3a\x0e\xc4\xe1\xff"
-  "\x67\x76\xd3\xde\x93\xbc\x11\x02\xef\xbb\x1b\x18\x52\x32\x03\x07"
-  "\xf0\xea\xb1\xfa\x36\x70\x33\x02\x20\x3f\x92\xec\x0c\xbe\xc6\xd5"
-  "\xe8\x57\x92\x43\xe4\x3e\x4a\xdd\xd4\xd0\x8c\x7b\x6c\x02\x6c\xfd"
-  "\x1e\x8f\x84\x34\x2f\xdf\x81\xe1\x36";
+  "\x30\x82\x01\x79\x30\x82\x01\x1f\x02\x01\x01\x30\x0a\x06\x08\x2a"
+  "\x86\x48\xce\x3d\x04\x03\x02\x30\x3c\x31\x11\x30\x0f\x06\x03\x55"
+  "\x04\x03\x0c\x08\x53\x6f\x66\x74\x20\x55\x32\x46\x31\x14\x30\x12"
+  "\x06\x03\x55\x04\x0a\x0c\x0b\x47\x69\x74\x48\x75\x62\x20\x49\x6e"
+  "\x63\x2e\x31\x11\x30\x0f\x06\x03\x55\x04\x0b\x0c\x08\x53\x65\x63"
+  "\x75\x72\x69\x74\x79\x30\x1e\x17\x0d\x31\x37\x30\x37\x32\x34\x32"
+  "\x33\x33\x38\x32\x37\x5a\x17\x0d\x32\x37\x30\x37\x32\x32\x32\x33"
+  "\x33\x38\x32\x37\x5a\x30\x3c\x31\x11\x30\x0f\x06\x03\x55\x04\x03"
+  "\x0c\x08\x53\x6f\x66\x74\x20\x55\x32\x46\x31\x14\x30\x12\x06\x03"
+  "\x55\x04\x0a\x0c\x0b\x47\x69\x74\x48\x75\x62\x20\x49\x6e\x63\x2e"
+  "\x31\x11\x30\x0f\x06\x03\x55\x04\x0b\x0c\x08\x53\x65\x63\x75\x72"
+  "\x69\x74\x79\x30\x59\x30\x13\x06\x07\x2a\x86\x48\xce\x3d\x02\x01"
+  "\x06\x08\x2a\x86\x48\xce\x3d\x03\x01\x07\x03\x42\x00\x04\xf6\x9c"
+  "\xab\x24\x14\x4b\xb4\xef\x87\xf7\x0f\x23\x1c\x5c\xd4\xf5\x78\x04"
+  "\xac\xf8\xe0\xc6\xb2\xb3\xe3\x52\x18\x3d\x80\x39\x1f\x6b\xd2\x79"
+  "\xd2\x6a\x4c\x83\x64\x74\xe6\xc2\xda\x23\x93\xff\xac\x1d\x50\x34"
+  "\x6c\x5c\x23\x90\x65\x57\x93\x3e\xcb\x93\xff\x6e\xde\xd1\xa3\x17"
+  "\x30\x15\x30\x13\x06\x0b\x2b\x06\x01\x04\x01\x82\xe5\x1c\x02\x01"
+  "\x01\x04\x04\x03\x02\x03\x08\x30\x0a\x06\x08\x2a\x86\x48\xce\x3d"
+  "\x04\x03\x02\x03\x48\x00\x30\x45\x02\x21\x00\xf6\xc8\xfd\x40\xc2"
+  "\x32\xb4\xd2\xa6\x15\x24\x10\x44\x5f\xb0\x78\x6c\x63\x69\x74\xd4"
+  "\xfd\xf0\x53\xc5\x46\xbe\xc9\x9a\xc7\x02\x67\x02\x20\x40\xb5\xf1"
+  "\xa4\x5f\xc7\xfc\x06\xd4\x88\x9d\xde\xfb\x8c\xdc\x23\xba\xb0\x42"
+  "\xd9\xf0\x83\x3d\xfd\x06\x29\x7f\x07\x9d\x81\xf1\x8a";
```

Comment:
Does it make sense to dump the text output of this cert in a comment to make it clearer as to what's going on?